src/native/libs/build-native.proj

```diff
@@ -78,11 +78,15 @@
           BeforeTargets="Build"
           Condition="$([MSBuild]::IsOsPlatform(Windows))">
     <PropertyGroup>
-        <_BuildNativeArgs Condition="'$(Ninja)' == 'false'">$(_BuildNativeArgs) msbuild</_BuildNativeArgs>
+      <_BuildNativeArgs Condition="'$(Ninja)' == 'false'">$(_BuildNativeArgs) msbuild</_BuildNativeArgs>
+      <DOTNET_EMSCRIPTEN_LLVM_ROOT>$(EMSDK_PATH)\bin</DOTNET_EMSCRIPTEN_LLVM_ROOT>
+      <DOTNET_EMSCRIPTEN_NODE_JS>$(NuGetPackageRoot)runtime.$(_portableHostOS)-$(BuildArchitecture).microsoft.netcore.runtime.wasm.node.transport/$(NodePackageVersion)/tools/$(_portableHostOS)-$(BuildArchitecture)/bin/node</DOTNET_EMSCRIPTEN_NODE_JS>
+      <_BuildNativeEnvironmentVariables>$(_BuildNativeEnvironmentVariables);DOTNET_EMSCRIPTEN_LLVM_ROOT=$(DOTNET_EMSCRIPTEN_LLVM_ROOT);EM_LLVM_ROOT=$(DOTNET_EMSCRIPTEN_LLVM_ROOT);DOTNET_EMSCRIPTEN_NODE_JS=$(DOTNET_EMSCRIPTEN_NODE_JS);DOTNET_EMSCRIPTEN_BINARYEN_ROOT=$(EMSDK_PATH)</_BuildNativeEnvironmentVariables>
     </PropertyGroup>
     <!-- Run script that uses CMake to generate and build the native files. -->
     <Message Text="&quot;$(MSBuildThisFileDirectory)build-native.cmd&quot; $(_BuildNativeArgs)" Importance="High"/>
-    <Exec Command="&quot;$(MSBuildThisFileDirectory)build-native.cmd&quot; $(_BuildNativeArgs)" EnvironmentVariables="$(_BuildNativeEnvironmentVariables)" />
+    <Message Text="_BuildNativeEnvironmentVariables = $(_BuildNativeEnvironmentVariables)" Importance="High"/>
```

Comment:
This prints:
```
  _BuildNativeEnvironmentVariables = ;DOTNET_EMSCRIPTEN_LLVM_ROOT=\bin;EM_LLVM_ROOT=\bin;DOTNET_EMSCRIPTEN_NODE_JS=D:\a\_work\1\s\.packages\runtime.win-x64.microsoft.netcore.runtime.wasm.node.transport/9.0.0-alpha.1.24175.1/tools/win-x64/bin/node;DOTNET_EMSCRIPTEN_BINARYEN_ROOT=
```
@radekdoulik, do you have an idea how to make `EMSDK_PATH` not empty during the native build?